blueocean-dashboard/src/main/js/Capabilities.js

```diff
@@ -3,3 +3,4 @@ export const SIMPLE_PIPELINE = 'org.jenkinsci.plugins.workflow.job.WorkflowJob';
 export const MATRIX_PIPELINE = 'hudson.matrix.MatrixProject';
 export const FREESTYLE_JOB = 'hudson.model.FreeStyleBuild';
 export const PIPELINE_JOB = 'org.jenkinsci.plugins.workflow.job.WorkflowRun';
+export const DISABLEABLE = 'io.jenkins.blueocean.rest.model.Disableable';
```

Comment:
since we didn't end up using the capability, i'm wondering if I should delete the code for it. It was only attached to freestyle, but apparently more job types support it.

Reply:
Yea I think I will